package.json

```diff
@@ -32,10 +32,10 @@
         "sharp": "^0.33.5"
     },
     "devDependencies": {
-        "@appwrite.io/console": "^0.6.4",
+        "@appwrite.io/console": "https://pkg.pr.new/appwrite/appwrite/@appwrite.io/console@7773939",
```

Comment:
@ItzNotABug please let me know what we should update this to at the time of release?

Reply:
fine for now.